tests/test_process.py

```diff
@@ -129,50 +129,30 @@ def test_get_lc2_path():
 
 @responses.activate
 def test_get_s2_metadata_not_found():
-    responses.add(responses.GET, f'{process.S2_SEARCH_URL}/foo', status=404)
-    responses.add(
-        responses.POST, process.S2_SEARCH_URL,
-        body='{"numberReturned": 0}', status=200,
-    )
+    responses.add(responses.GET, f'{process.S2_GRANULE_DIR}////foo.SAFE/manifest.safe', status=404)
     with pytest.raises(ValueError):
         process.get_s2_metadata('foo')
 
 
 @responses.activate
-def test_get_s2_metadata_cog_id():
-    responses.add(
-        responses.GET, f'{process.S2_SEARCH_URL}/2FS2B_22WEB_20200913_0_L2A',
-        body='{"foo": "bar"}', status=200,
-    )
-
-    assert process.get_s2_metadata('2FS2B_22WEB_20200913_0_L2A') == {'foo': 'bar'}
+def test_get_s2_metadata():
+    # TODO implement me
+    pass
 
 
-@responses.activate
-def test_get_s2_metadata_esa_id():
-    responses.add(
-        responses.GET, f'{process.S2_SEARCH_URL}/S2B_MSIL2A_20200913T151809_N0214_R068_T22WEB_20200913T180530',
-        status=404,
-    )
-    responses.add(
-        responses.POST, process.S2_SEARCH_URL,
-        body='{"numberReturned": 1, "features": [{"foo": "bar"}]}', status=200,
-    )
+def test_get_s2_path(test_data_directory):
+    scene_name = 'S2A_MSIL1C_20160616T112217_N0204_R137_T29QKF_20160617T193500'
+    f = open(f'{test_data_directory}/{scene_name}.manifest.safe', 'r')
+    manifest_text = f.read()
+    path = process.get_s2_path(manifest_text, scene_name)
+    assert path == '/vsicurl/https://storage.googleapis.com/gcp-public-data-sentinel-2/tiles//./GRANULE' \
+                   '/S2A_OPER_MSI_L1C_TL_SGS__20160616T181414_A005139_T29QKF_N02.04/IMG_DATA' \
+                   '/S2A_OPER_MSI_L1C_TL_SGS__20160616T181414_A005139_T29QKF_B08.jp2'
```

Comment:
This covers the pre-2016-12-06 case. Scenes after 2016-12-06 have a difference manifest.safe format, so we should copy/paste a similar test for `S2B_MSIL1C_20200419T060719_N0209_R105_T38EMQ_20200419T091056.manifest.safe` to test the more recent format.